app/views/flexible_page/flexible_sections/_feed.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="govuk-grid-column-two-thirds" data-flexible-section="feed_document_list">
```

Comment:
About the data attributes - since we know what's going to be in this particular bit of the page (so far) it might be enough to put one data attribute wrapping the whole section, and base the tests off that. For example the test for the 'see all' link - we know there should be a link with that href and text somewhere in here, I think it's enough that it's within this section rather than more granular.

Reply:
Thanks, I've simplified it now 👍